bdb/column.go

```diff
@@ -29,6 +29,11 @@ type Column struct {
 	// tinyint(1) instead of tinyint
 	// Used for "tinyint-as-bool" flag
 	FullDBType string
+
+	// MS SQL only bits
+	// Used to indicate that the value
+	// for this column is auto generated by database on insert (i.e. - timestamp (old) or rowversion (new))
+	AutoGenerated bool
```

Comment:
Fields with default values can be explicitly inserted or updated, but fields of type rowversion, timestamp - can not.
According to the description of transact-sql (see https://technet.microsoft.com/en-us/library/ms182776(v=sql.110).aspx) those fileds totally controlled by database:

> Every time that a row with a rowversion column is modified or inserted, the incremented database rowversion value is inserted in the rowversion column.

If we try to update or insert data into this column, we will receive an error message.